commit 9fb1a1903345fea598f48277576a3589a972b72e
Author: Dave Jiang <dave.jiang@intel.com>
Date:   Mon Oct 30 13:22:20 2017 -0700

    nfit_test: add error injection DSMs
    
    Add nfit_test emulation for the new ACPI 6.2 error injectino DSMs.
    This will allow unit tests to selectively inject the errors they wish to
    test for.
    
    Signed-off-by: Dave Jiang <dave.jiang@intel.com>
    [vishal: Move injection functions to ND_CMD_CALL]
    [vishal: Add support for the notification option]
    [vishal: move an nfit_test private definition into a local header]
    Signed-off-by: Vishal Verma <vishal.l.verma@intel.com>
    Signed-off-by: Dan Williams <dan.j.williams@intel.com>

diff --git a/tools/testing/nvdimm/test/nfit_test.h b/tools/testing/nvdimm/test/nfit_test.h
index 52c83be9dcfa..113b44675a71 100644
--- a/tools/testing/nvdimm/test/nfit_test.h
+++ b/tools/testing/nvdimm/test/nfit_test.h
@@ -33,6 +33,11 @@ struct nfit_test_resource {
 };
 
 #define ND_TRANSLATE_SPA_STATUS_INVALID_SPA  2
+#define NFIT_ARS_INJECT_INVALID 2
+
+enum err_inj_options {
+	ND_ARS_ERR_INJ_OPT_NOTIFY = 0,
+};
 
 /* nfit commands */
 enum nfit_cmd_num {